src/gui/torrentmodel.cpp

```diff
@@ -55,10 +56,33 @@ static QIcon getErrorIcon();
 
 static bool isDarkTheme();
 
+static bool isTextVisible(const QModelIndex& idx, QObject *widget, const BitTorrent::TorrentHandle *torrent)
```

Comment:
The function name is misleading to me. Would prefer either `isTextFullyVisible()` or something like `doesTextFit()`